packages/cli/src/databases/entities/WorkflowEntity.ts

```diff
@@ -46,6 +46,13 @@ export class WorkflowEntity extends WithTimestampsAndStringId implements IWorkfl
 	})
 	staticData?: IDataObject;
 
+	@Column({
+		type: jsonColumnType,
+		nullable: true,
+		transformer: objectRetriever,
+	})
+	meta?: IDataObject;
```

Comment:
Can we type what meta can be? So when we get the record we can get the intelisense there, like we do on settings?

Reply:
sure